public/data.json

```diff
@@ -99,6 +99,16 @@
             "id": "1",
             "src": "carousel_img_1.png",
             "alt": "img1"
+        },
+        {
+            "id": "2",
+            "src": "carousel_img_2.png",
+            "alt": "img2"
+        },
+        {
+            "id": "3",
```

Comment:
I am not sure about ids in React, but in Vue you could use indexes for iterating thorough element. If this is the case for React then you should remove custom ids and use indexes instead

Reply:
Roman did not recommend using indexes if I remember correctly